.travis.yml

```diff
@@ -1,7 +1,4 @@
 language: scala
-scala:
```

Comment:
Since we're specifying `script`, we don't need to specify the Scala version here. SBT will take care of testing with the different Scala versions according to the build configuration.

Reply:
Ok

Reply:
Ok